test/run_pass/test_net_10.js

```diff
@@ -39,6 +41,11 @@ socket1.on('connect', function() {
 });
 
 process.on('exit', function() {
-  assert(timedout);
+  if (process.platform == 'nuttx' || process.platform == 'tizenrt') {
```

Comment:
Looks strange. I believe we should be able to execute the same code as possible on all supported platforms. Is this an inevitable problem?